udata/core/dataset/csv.py

```diff
@@ -33,6 +33,7 @@ class DatasetCsvAdapter(csv.Adapter):
         ('tags', lambda o: ','.join(o.tags)),
         ('archived', lambda o: o.archived or False),
         ('resources_count', lambda o: len(o.resources)),
+        'downloads',
```

Comment:
I wonder if naming it `resources_downloads` would make it more explicit, following `resources_count`?

Reply:
I use the term requested in the issue https://github.com/etalab/data.gouv.fr/issues/1237 but I can change it.

Reply:
Let's keep with `downloads` :+1: